File: day21Generics.cs
using System;

public class Solution {
  //above has been written
  // Add your code here
   static void printArray<Element>(Element[] array)
       {
         foreach( var element in array)
             {
             Console.WriteLine(element);
         }
   }
  //below has been written
  static void Main(string[] args) {
      int[] vInt = new int[] {1, 2, 3};
      string[] vString = new string[] {"Hello", "World"};

      printArray<int>(vInt);
      printArray<string>(vString);
  }
}

File: day21Generics.cpp
#include <iostream>
#include <vector>

using namespace std;
//Above is already be written
// Add your code here
template<class Element>
    void printArray(vector<Element> arr) {
    for (int i=0; i<arr.size(); i++)
        cout<<arr[i]<<endl;
}
//below has been written already
int main() {

    vector<int> vInt{1, 2, 3};
    vector<string> vString{"Hello", "World"};

    printArray<int>(vInt);
    printArray<string>(vString);

    return 0;
}
